__tests__/habits/habits.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-underscore-dangle, import/order */
+const mongoose = require('mongoose');
+const { app } = require('../../server');
+const request = require('supertest').agent(app);
+
+const habit = {
+	name: 'Test habit',
+	user: '5c0cee8d8c452f13fbfe2281',
+	description: 'This is a habit',
+	tags: ['office', 'work', 'omg'],
+	type: 'Positive',
+	frequency: 'Daily',
```

Comment:
Do we want more fine-tuned frequency though? We could make this an object with 'times' and 'period' inside it. If a user wants to instil a habit that occurs 3 times a week it could be `frequency: { times: 3, period: 'week' }`

Reply:
This can all be changed in another PR

Reply:
Sure, that's not that much related with the purpose of this PR. A new issue regarding this would be great.